flake.nix

```diff
@@ -15,15 +15,24 @@
       flake-compat.follows = "flake-compat";
       flake-utils.follows = "flake-utils";
     };
+    nixgl.url = "github:guibou/nixGL";
+    nixgl.inputs = {
+      nixpkgs.follows = "nixpkgs";
+    };
   };
 
-  outputs = { self, nixpkgs_latest, nixpkgs, nix-direnv, flake-utils, ... }:
+  outputs = { self, nixpkgs_latest, nixpkgs, nixgl, nix-direnv, flake-utils, ... }:
     flake-utils.lib.eachDefaultSystem (system:
       let
-        overlays = [ nix-direnv.overlay ];
+        # import <XXX> underneath this project
```

Comment:
As discussed. Ideally, we should use direnv hook to intercept evaluation of `projects/xxx/shell.nix`. This way we could introduce a shim nix expression for injecting overlays defined in `flake.nix`.